packages/accounts-base/accounts_client.js

```diff
@@ -63,6 +66,45 @@ export class AccountsClient extends AccountsCommon {
   }
 
   /**
+   * @summary Register a new login function on client. Intended for OAuth package authors. You can call the login function by using `Accounts.callLoginFunction`.
```

Comment:
With the `applyLoginFunction` change mentioned below, I believe these comments should become:

```
* @summary Register a new login function on the client. Intended for OAuth package authors. You can call the login function by using `Accounts.callLoginFunction` or `Accounts.callLoginFunction`.
...
* @param {String} funcName The name of your login function. Used by `Accounts.callLoginFunction` and `Accounts.applyLoginFunction`. Should be the OAuth provider name accordingly.
```